lib/features/post/widgets/form_widget/handlers.dart

```diff
@@ -59,6 +60,7 @@ extension FormWidgetHandlers on FormWidget {
                 file: form.file,
               );
           context.read<PostFormCubit>().setVisible(false);
+          context.read<ThreadsCubit>().getThreads(board, sort!);
```

Comment:
sort is a nullable property on line 37, then you are forcing non null here. Either make Sort required or provide a default if null.